src/runtime/runtime.js

```diff
@@ -709,6 +721,29 @@ export class ConfiguredRuntime {
   /** @override */
   showOffers(opt_options) {
     return this.documentParsed_.then(() => {
+      if (opt_options['oldSku']) {
+        console.error(
+          'Use the showUpdateOffers method for subscription updates'
```

Comment:
const errorMessage = 
'The showOffers() method cannot be used to update a subscription. Please use the showUpdateOffers() method instead.';

console.error(errorMessage);
return new Error(errorMessage);

assign it to a const